<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="1" column="1">
             <widget class="KComboBox" name="distroIntervalCB"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="intervalL">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Check for new updates:</string>
              </property>
              <property name="buddy">
               <cstring>intervalCB</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KComboBox" name="intervalCB">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0" rowspan="3">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="checkUpdatesMobileCB">
                <property name="text">
                 <string>when on a mobile broadband network connection</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="checkUpdatesBatteryCB">
                <property name="text">
                 <string>when running on battery</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="autoInsL">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Automatically update:</string>
              </property>
              <property name="buddy">
               <cstring>autoCB</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="KComboBox" name="autoCB">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0" rowspan="2">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="installUpdatesBatteryCB">
                <property name="text">
                 <string>when running on battery</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="installUpdatesMobileCB">
                <property name="text">
                 <string>when on a mobile broadband network connection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QCheckBox" name="appLauncherCB">
              <property name="text">
               <string>Show application launcher after installing new applications</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0" colspan="2">
             <widget class="QCheckBox" name="autoConfirmCB">
              <property name="text">
               <string>Show confirmation dialog when installing or updating packages</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Check for distribution upgrades:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="originTV">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="showOriginsCB">
         <property name="text">
          <string>&amp;Show origins of debug and development packages</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="editOriginsPB">
         <property name="text">
          <string>Edit Origins</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="KMessageWidget" name="messageWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header>KMessageWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
